array-manipulation-2/src/take.js

```diff
@@ -0,0 +1,13 @@
+/* exported take */
+function take(array, count) {
+  var newArray = [];
+
+  for (var i = 0; i < count; i++) {
+    var val = array[i];
+    if (val || typeof val === 'boolean') {
```

Comment:
Can be removed 👁️ 👁️

Reply:
If this is to deal with the array being shorter than the length requested, I would use a conditional before the loop to check if the count is larger than the length, and if it is, just return an empty array